### ISSUE TITLE
Tag auto complete is broken

### ISSUE THREAD

**Issue by chmac** [dendronhq/dendron#2724]
## Please select the area your bug applies to. (Multiple selections are Possible. You can leave blank if you're not sure.)

- [x] Workspace.  VSCode workspace, vaults, Intellisense/autocomplete, Dendron settings
- [ ] Lookup.  Dendron's Lookup Command
- [ ] Views.  Dendron Preview, Tree View, Side Panels in the UI
- [ ] Schema.  Dendron Schemas
- [ ] Pod.  Data import from / export to Dendron
- [ ] Publish.  External Site Publish
- [ ] Markdown.  Markdown features, such as syntax support and features

## Describe the bug

Pressing ctrl+space no longer auto completes tags.

## To Reproduce

1. Create a tag foo
2. Open a note
3. Type `#f` and press ctrl-space

## Expected behavior

This should present a list of tags to auto complete (intellisense?).

## Desktop (please complete the following information)

- OS: macOS
- Versions (all that apply to bug):
  - Dendron Extension: v0.89.0
  - VS Code version: 1.66.1
  - Dendron CLI:

## Additional context

This seems to have only started happening today, maybe since I upgraded VSCode?

Please let me know if I can help to debug this in some way.

EDIT: I originally wrote ctrl+enter, but this was an error, I meant ctrl+space, sorry about the confusion.

**Comment by chmac** [comment]
Never mind, turns out my keyboard shortcut for this was changed, it's an error in my setup. Closing this.

**Comment by chmac** [comment]
Hmm, nope, it's not that simple. The ctrl+enter shortcut is still enabled, nothing has changed there, just that some other shortcuts work, but somehow in dendron ctrl+enter doesn't send the "trigger suggest" shortcut.

Here's the log output of the keyboard shortcut debugger when I use ctrl-enter first (doesn't work)

```
[2022-04-10 12:31:55.883] [renderer1] [info] [KeybindingService]: / Received  keydown event - modifiers: [ctrl], code: ControlLeft, keyCode: 17, key: Control
[2022-04-10 12:31:55.883] [renderer1] [info] [KeybindingService]: | Converted keydown event - modifiers: [ctrl], code: ControlLeft, keyCode: 5 ('Ctrl')
[2022-04-10 12:31:55.884] [renderer1] [info] [KeybindingService]: \ Keyboard event cannot be dispatched in keydown phase.
[2022-04-10 12:31:56.135] [renderer1] [info] [KeybindingService]: + Storing single modifier for possible chord ctrl.
[2022-04-10 12:31:56.438] [renderer1] [info] [KeybindingService]: + Clearing single modifier due to 300ms elapsed.
```

Then option+escape (does work):

```
[2022-04-10 12:32:40.183] [renderer1] [info] [KeybindingService]: / Received  keydown event - modifiers: [alt], code: AltLeft, keyCode: 18, key: Alt
[2022-04-10 12:32:40.183] [renderer1] [info] [KeybindingService]: | Converted keydown event - modifiers: [alt], code: AltLeft, keyCode: 6 ('Alt')
[2022-04-10 12:32:40.183] [renderer1] [info] [KeybindingService]: \ Keyboard event cannot be dispatched in keydown phase.
[2022-04-10 12:32:40.655] [renderer1] [info] [KeybindingService]: / Received  keydown event - modifiers: [alt], code: Escape, keyCode: 27, key: Escape
[2022-04-10 12:32:40.656] [renderer1] [info] [KeybindingService]: | Converted keydown event - modifiers: [alt], code: Escape, keyCode: 9 ('Escape')
[2022-04-10 12:32:40.656] [renderer1] [info] [KeybindingService]: | Resolving alt+[Escape]
[2022-04-10 12:32:40.656] [renderer1] [info] [KeybindingService]: \ From 1 keybinding entries, matched editor.action.triggerSuggest, when: editorHasCompletionItemProvider && textInputFocus && !editorReadonly, source: built-in.
[2022-04-10 12:32:40.954] [renderer1] [info] [KeybindingService]: + Ignoring single modifier alt due to it being pressed together with other keys.
```

**Comment by aleksey-rowan** [comment]
Hey, @chmac, for me `Ctrl + Enter` creates a note for the tag, doesn't auto-complete. If I have `#foo` tag and press `Ctrl + Enter` after typing `#f`, a new tag of `#f` will be created. I think this behaviour was standard since tags were introduced. 

If you the following settings in your workspace pressing `Tab` should auto-complete the tag to the first match:
```js
"editor.suggest.snippetsPreventQuickSuggestions": false,
"editor.suggest.showSnippets": true,
"editor.tabCompletion": "on",
```

To show the list of available tags, I always press `Ctrl + Space` and I also think this is a standard VSCode shortcut.

There is already a similar issue open. Could you please check if it addresses the same problem?

- https://github.com/dendronhq/dendron/issues/1518

**Comment by chmac** [comment]
@aleksey-rowan Ah shoot, sorry, you're right, I mean ctrl+space, but I wrote ctrl+enter, I'll edit my original message now. Sorry for the confusion.

I've looked at #1518 but I'm not sure if it's the same issue. I can't quite follow that issue precisely. But I think it's different. I can get the behaviour to activate correctly (the auto suggest) by using a different keyboard shortcut like alt+escape. But not any more with ctrl+space. I used ctrl+space until the most recent VSCode upgrade I think. I use this feature a lot, and only noticed the problem today, right after a VSCode upgrade.

**Comment by aleksey-rowan** [comment]
@chmac, oh, okay, so it is `Ctrl + Space` problem. I'm using VSCode 1.66.1 and this trigger works for me just fine. Does `Ctrl + Space` no longer work in Dendron workspace only or everywhere else as well? Try opening a random `.json` (outside of any workspace) file and check if `Ctrl + Space` brings up any suggestions.

Also, a silly question, but have you tried closing VSCode completely (or rebooting your computer)?

**Comment by chmac** [comment]
@aleksey-rowan Thanks for the help. I've just now figured out that there was a system wide shortcut which was capturing ctrl+space. When I activated a new keyboard layout, ctrl+space became a shortcut to trigger the next layout. I hadn't specifically enabled it so I wasn't aware of the new shortcut, and it wasn't doing anything because I disabled the extra layout and so I only had 1.

Again, sorry about the spurious bug report, thanks for the pointers which helped me figure it out.
